BraintreeModels/src/main/java/com/braintreepayments/api/PaymentMethod.java

```diff
@@ -46,120 +43,49 @@ void setSource(String source) {
+        return new MetadataBuilder()
+                .sessionId(sessionId)
+                .source(source)
+                .integration(integration)
+                .build();
     }
 
-    /**
-     * @param authorization The current authorization being used.
-     * @return String representation of a GraphQL request for {@link PaymentMethodNonce}.
-     * @throws BraintreeException Thrown if resources cannot be accessed.
-     */
-    public String buildGraphQL(Authorization authorization) throws BraintreeException {
+
+    JSONObject buildJSON() throws JSONException {
         JSONObject base = new JSONObject();
-        JSONObject input = new JSONObject();
-        JSONObject variables = new JSONObject();
-
-        try {
-            base.put(GRAPHQL_CLIENT_SDK_METADATA_KEY, new MetadataBuilder()
-                    .sessionId(sessionId)
-                    .source(source)
-                    .integration(integration)
-                    .build());
-
-            JSONObject optionsJson = new JSONObject();
-            if (validateSet) {
-                optionsJson.put(VALIDATE_KEY, validate);
-            } else {
-                if (authorization instanceof ClientToken) {
-                    optionsJson.put(VALIDATE_KEY, true);
-                } else if (authorization instanceof TokenizationKey) {
-                    optionsJson.put(VALIDATE_KEY, false);
-                }
-            }
-            input.put(OPTIONS_KEY, optionsJson);
-            variables.put(Keys.INPUT, input);
-
-            buildGraphQL(base, variables);
-
-            base.put(Keys.VARIABLES, variables);
-        } catch (JSONException ignored) {}
-
-        return base.toString();
+        base.put(MetadataBuilder.META_KEY, buildMetadataJSON());
+        return base;
     }
 
-    protected PaymentMethod(Parcel in) {
+    PaymentMethod(Parcel in) {
         integration = in.readString();
         source = in.readString();
-        validate = in.readByte() > 0;
-        validateSet = in.readByte() > 0;
         sessionId = in.readString();
     }
 
-    public void writeToParcel(Parcel dest, int flags) {
+    void writeToParcel(Parcel dest, int flags) {
         dest.writeString(integration);
         dest.writeString(source);
-        dest.writeByte(validate ? (byte) 1 : 0);
-        dest.writeByte(validateSet ? (byte) 1 : 0);
         dest.writeString(sessionId);
     }
 
-    protected String getDefaultSource() {
+    String getDefaultSource() {
         return "form";
     }
 
-    protected String getDefaultIntegration() {
+    String getDefaultIntegration() {
         return "custom";
     }
 
-    protected abstract void buildJSON(JSONObject base, JSONObject paymentMethodNonceJson) throws JSONException;
-
-    protected abstract void buildGraphQL(JSONObject base, JSONObject input) throws BraintreeException,
-            JSONException;
-
-    public abstract String getApiPath();
-
-    public abstract String getResponsePaymentMethodType();
```

Comment:
This method was not being used.

Reply:
Is it something that merchants might have used? Or maybe Drop-in?

Reply:
It's not used by Drop-in, and it's not currently documented anywhere in the Braintree Dev Docs. We figured that it was safe to make package-private for now (and later potentially remove it without a breaking change), and it could be added back if any merchants do need it.